src/rdkafka_partition.c

```diff
@@ -609,26 +609,23 @@ void rd_kafka_toppar_desired_del (rd_kafka_toppar_t *rktp) {
  * Append message at tail of 'rktp' message queue.
  */
 void rd_kafka_toppar_enq_msg (rd_kafka_toppar_t *rktp, rd_kafka_msg_t *rkm) {
+	int wakeup_fd, queue_len;
+	rd_kafka_t *rk;
```

Comment:
You dont need to keep `rk`, use rd_assert() which does not take an rk argument.

Reply:
ah thanks, though this may conflict with your comments on the logging change below?